templates/solidity-frameworks/foundry/packages/foundry/foundry.toml.template.mjs

```diff
@@ -0,0 +1,58 @@
+import { withDefaults } from "../../../../utils.js";
+
+const content = ({ profileDefaults, rpcEndpoints, etherscan, fmt, extraConfig }) => `[profile.default]
```

Comment:
nit (could be ignored): variable names could be more detailed, like `extraProfileDefaults, extraRpcEndpoints, extraEthercsanConfig, extraFormattingConfig, extraConfig`

Reply:
yup makes sense! Thanks updated 🙌